angular/lib/src/config.ts

```diff
@@ -0,0 +1,41 @@
+import type {
+	AdaptPropsSlots,
+	AngularWidget,
+	Partial2Levels,
+	Widget,
+	WidgetFactory,
+	WidgetProps,
+	WidgetsConfigStore,
+} from '@agnos-ui/angular-headless';
+import {widgetsConfigFactory} from '@agnos-ui/angular-headless';
+import type {WidgetsConfig as CoreWidgetsConfig} from '@agnos-ui/style-bootstrap/config';
```

Comment:
Is it really `CoreWidgetsConfig`? These are the exported ones by `style-bootstrap`.
```suggestion
import type {WidgetsConfig as BoostrapWidgetsConfig} from '@agnos-ui/style-bootstrap/config';
```